README.md

```diff
@@ -119,184 +133,16 @@ ddn auth login
-
-Let's start our connector's Docker Compose file by running the following from inside the connector's subgraph:
-
-```bash
-docker compose -f compose.yaml up
-```
-
-  <!-- TODO: As before, update <CONNECTOR_NAME> to match step 2 -->
-
-### Step 9. Update the new `DataConnectorLink` object
-
-Finally, now that our `DataConnectorLink` has the correct environment variables configured for the connector,
-we can run the update command to have the CLI look at the configuration JSON and transform it to reflect our database's
-schema in `hml` format. In a new terminal tab, run:
-
-```bash
-ddn connector-link update <connector-name> --subgraph my_subgraph/subgraph.yaml --env-file my_subgraph/.env.my_subgraph.local
-```
-
-After this command runs, you can open your `my_subgraph/metadata/<connector-name>.hml` file and see your metadata completely
-scaffolded out for you 🎉
-
-### Step 10. [Connector-specific steps]
-
-<!-- TODO: If there are connector-specific configuration steps, such as adding env vars or anything of the like, add
-these steps here. -->
-
-### Step 11. Deploy your connector
-
-Connectors can be deployed to Hasura DDN or, if you choose, your own infrastructure.
-
-#### Deploy [ConnectorName] to Hasura DDN
-
-Prepare a `env.cloud` file for the connector. This should utilize whichever environment variables, such as a connection
-string, you intend to use with your deployed supergraph. As an example:
-
-```env
-CONNECTION_URI=<database-connection-uri>
-```
-
-Then, run the following command taking care to update the referenced paths to match your project structure:
-
-```sh
-ddn connector build create \
-  --connector my_subgraph/connector/my_connector/connector.cloud.yaml \
-  --target-env-file my_subgraph/.env.my_subgraph.cloud \
-  --target-subgraph my_subgraph/subgraph.yaml \
-  --target-connector-link my_connector
-```
-
-This will deploy your connector and update your project's metadata.
-
-#### Deploy [ConnectorName] to your own infrastructure
-
-As a connector is an HTTP service built using Docker, you can host your connectors anywhere Docker containers can be
-hosted.
-
-On your own self- or cloud-hosted infrastructure, create a new build from within the connector's directory:
-
-```sh
-docker compose build
-```
-
-Then, bring the connector up:
-
-```sh
-docker compose up -d
-```
-
-And verify the connector is running:
-
-```sh
-docker ps
-```
-
-You'll need to follow whatever steps are necessary to expose your connector's port so that Hasura DDN can connect to it.
-Additionally, you'll need to update any cloud environment variables that your deployed supergraph needs from this
-connector (e.g., `MY_SUBGRAPH_MY_CONNECTOR_READ_URL`) to match your deployed connector's endpoint.
-
-## Example
+### Step n: Do more things...
+<!-- TODO: Document your connector well :) 
+See the documentation for the getting started with DDN [here](https://hasura.io/docs/3.0/getting-started/overview), 
+which can help you author these steps.
+-->
```

Comment:
We made a strong effort in earlier versions to call out the specific steps. At a minimum, we should stub out the CLI commands to initialize a new connector. 

From there, the instructions for the author should reflect connector-specific steps (e.g., environment variables) that need to be completed as a user is going through the wizard. It would be helpful if this README asked authors to include example values for these and where they can find the information either in their source's self-hosted config, or in the case of a cloud service, their dashboard, etc.

Reply:
I can improve this for the **_connector user_**, showing more of what is in the [quickstart docs](https://hasura.io/docs/3.0/getting-started/quickstart) for when the user runs `ddn connector init -i`, but it's going to be more or less a copy of what is there. I can't get more specific because we don't know what the connector author may enable in terms of env vars. 

For the **_connector author_** I don't know what to tell them regarding setting up the connector to integrate with the interactive mode on the connector init command; how to handle their environment variables etc. Maybe @praveenweb or @SandeepSamba or @varundey could help with this? 

Also, how do they integrate supply a default, demo, read-only source for their connector.

Reply:
Yep — I think, with the plugin available, all we have to do here is instruct the **author** to call out what a **user** can expect to see when they're running through the wizard. 

Of course, all will start by asking for a connector name and a port, but from there, the author should call out what else a user can expect to encounter and where they can find these values.

Reply:
OpenAPI connector already has this implemented. Maybe we can take inspiration from there